on-boot-script/dpkg-build-files/debian/postinst

```diff
@@ -17,13 +17,12 @@ set -e
 # for details, see https://www.debian.org/doc/debian-policy/ or
 # the debian-policy package
 
+alias deb-systemd-invoke='systemctl --no-block --'
 
 case "$1" in
     configure)
        scp -P "$(cat /etc/unifi-os/ssh_proxy_port)" -o StrictHostKeyChecking=no -q /usr/share/udm-boot/on_boot.sh root@localhost:/mnt/data/on_boot.sh
        /sbin/ssh-proxy 'chmod +x /mnt/data/on_boot.sh && mkdir -p /mnt/data/on_boot.d'
-
-       systemctl enable --now udm-boot
```

Comment:
You're setting an alias, but you don't seem to be calling the alias, so the onboot service isn't being enabled or started on installation.

Am I missing something?

Reply:
Yes, you missed it.

```
#DEBHELPER#
```

Will be replaced by a large chunk of code, which include this line:

```
                deb-systemd-invoke $_dh_action 'udm-boot.service' >/dev/null || true
```

Reply:
Ah, that's a macro not a comment. Woops.